### ISSUE TITLE
Failure to update standard EN deck 

### ISSUE THREAD

**Issue by sortega** [axelboc/anki-ultimate-geography#392]
I've installed Anki and CrowdAnki in their latest versions in a brand new airbook and importing from the uncompressed zip I get this error:

```
Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue on the add-on support site.
Debug info:
Anki 2.1.40 (cf446733) Python 3.8.6 Qt 5.14.2 PyQt 5.14.2
Platform: Mac 10.16
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2021-02-08 07:57:47
Add-ons possibly involved: ⁨CrowdAnki JSON exportimport Edit history Collaborate on deck creation⁩

Caught exception:
Traceback (most recent call last):
  File "/Users/sortega/Library/Application Support/Anki2/addons21/1788670778/anki/ui/action_vendor.py", line 31, in <lambda>
    lambda: AnkiJsonImporter.import_deck(self.window.col, self.directory_vendor))
  File "/Users/sortega/Library/Application Support/Anki2/addons21/1788670778/importer/anki_importer.py", line 112, in import_deck
    AnkiJsonImporter.import_deck_from_path(collection, Path(directory_path))
  File "/Users/sortega/Library/Application Support/Anki2/addons21/1788670778/importer/anki_importer.py", line 101, in import_deck_from_path
    if importer.load_deck(directory_path):
  File "/Users/sortega/Library/Application Support/Anki2/addons21/1788670778/importer/anki_importer.py", line 36, in load_deck
    aqt.mw.backup()
  File "aqt/main.py", line 588, in backup
AssertionError
```

**Comment by aplaice** [comment]
Thanks for the report and sorry for the bug!

Please try Anki 2.1.39. AFAICT `CrowdAnki` isn't yet compatible with the just-released Anki 2.1.40.

**Comment by sortega** [comment]
Thanks, with Anki 2.1.39 works like a charm
